src/scripts/timeout.js

```diff
@@ -0,0 +1,15 @@
+// Goes to home page for idle timer
+function goHome() {
+  showPage("1");
+}
+
+// Set Timer
+function setIdleTimer(id) {
+    if (id && id != "1") {
```

Comment:
@mmeiers I think from what Ezequiel said the idle-timer library is safe to use. I am going to ask at my JS class if the method we currently use for the timeout is ok as that just uses native JS so it would be one less library to include in the build. 

I think a show previous page function would be good next. I guess what we are looking to achieve is a framework were we can just add in the stuff we talked about earlier such as closing cards and scrolling pages to the top on timeout - stuff like that would need to be added on a case by case basis, but having somewhere ready to put that would be awesome.

On thing I noticed is I don't think we need to include reference to the ids after you removed the other code - I think this may be more widget specific stuff.

Thanks!